--- pr1502-task05-FGAccelerations.cpp.orig	2026-09-19 01:44:55
+++ pr1502-task05-mutant-source/src/models/FGAccelerations.cpp	2026-09-19 01:45:48
@@ -263,7 +263,7 @@
       const LagrangeMultiplier* mj = multipliers[j];
       a[i*n+j] = DotProduct(mj->ForceJacobian, v1) + DotProduct(mj->MomentJacobian, v2);
       if (mi->Wheel && mi->Wheel == mj->Wheel)
-        a[i*n+j] += mi->WheelJacobian * mj->WheelJacobian * mi->Wheel->Jinv;
+        a[i*n+j] += (i == j ? 1. : -1.) * mi->WheelJacobian * mj->WheelJacobian * mi->Wheel->Jinv;
     }
   }
 
